cli/get.go

```diff
@@ -76,11 +76,17 @@ A role must be specified when using this command through the --role flag. You ma
 		if ok {
 			applicationID = account.ID
 			label = account.Name
+		} else {
+			account = &Account{}
+		}
+
+		if account.MostRecentRole != "" && roleName == "" {
```

Comment:
do we prefer a most recent role over a user configured default? I can see a case where you may cycle through roles and want the tool provided role to not be the last one.

Reply:
The ask was specifically the most recent role from users

Reply:
for now, this seems to work for our users